File: app/src/main/java/com/sourabh/android/rxretrorealm/rest/GithubApiClient.java
package com.sourabh.android.rxretrorealm.rest;

import com.sourabh.android.rxretrorealm.modal.Repo;

import java.util.List;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface GithubApiClient {
    @GET("users/{user}/repos")
    Observable<List<Repo>> getUserRepo(@Path("user") String username);
}

File: app/src/main/java/com/sourabh/android/rxretrorealm/rest/GithubApiClient.kt
package com.sourabh.android.rxretrorealm.rest

import com.sourabh.android.rxretrorealm.modal.Repo

import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.Path

interface GithubApiClient {
    @GET("users/{user}/repos")
    fun getUserRepo(@Path("user") username: String): Observable<List<Repo>>
}
